src/tests/utils/insert-tomorrow-expiration-day.ts

```diff
@@ -0,0 +1,16 @@
+/** create string of tomorrow  expiration day */
+export const formattedDate = (): string => {
+  const tomorrow = new Date();
+  const border = 19;
+
+
+  tomorrow.setDate(tomorrow.getDate() + 1);
+
+  const newExpirationDay: string = tomorrow.toISOString().slice(0, border)
+    .replace('T', ' ') + '+02';
+
+  return newExpirationDay;
+};
+// Export the newExpirationDay variable directly
+export const newExpirationDay: string = formattedDate();
```

Comment:
no need to export result of `formattedDate` execution. Just call the function in the place where you need to use the result